README.md

```diff
@@ -27,14 +27,19 @@ Commit your code regularly and meaningfully. This helps both you (in case you ev
 Be prepared to demonstrate your understanding of this week's concepts by answering questions on the following topics. You might prepare by writing down your own answers before hand.
 
 1. What is the DOM?
+Document Object Model. A data structure or tree that has nested child elements
 
 2. What is an event?
+every user interaction with a site
 
 3. What is an event listener?
+the tracking process and action of a specific event on a specific element 
```

Comment:
an event listener is a function that runs when the specified event is triggered for the object it is bound to